addons/website_slides/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    slide_channels_count = fields.Integer('Followed Course Count', compute='_compute_channels_count')
+    company_slide_channels_count = fields.Integer('Followed Course By Company Partners Count', compute='_compute_company_channels_count')
+
+    def _compute_channels_count(self):
```

Comment:
Once everyone agrees on computed field name, update the method name to _compute_field_name (_compute_slide_channel_count for example).

Don't forget the triggers, even for a non stored used in a form view it is better to give them